packages/sanity/src/structure/comments/plugin/inspector/CommentsInspector.tsx

```diff
@@ -53,7 +55,7 @@ export function CommentsInspector(props: DocumentInspectorProps) {
 
 function CommentsInspectorInner(props: DocumentInspectorProps) {
   const {onClose} = props
-
+  const commentsEnabled = useCommentsEnabled()
```

Comment:
Since we use this hook in the parent component (`CommentsInspector`) – can we remove this hook here and pass the "mode" value as a prop instead?

```tsx
<RootLayer>
  <CommentsInspectorInner {...props} mode={mode} />
</RootLayer>
```